From 6254c72040c4bcf8e3abda862750827994266b51 Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Wed, 17 Aug 2022 07:08:41 +0300
Subject: [PATCH] [SW-69581] gaudi3: ppboot: increase stack size

In the following patch sha384 is used to verify the preboot image
hash. This causes the stack to overflow.
Stack size is increased to accommodate the new code requirement.

Change-Id: Ie9f9bbd2c16f7e70f258851a8220c83527a1a1c0
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/arch/gaudi3/arc/ppboot.lds.S b/arch/gaudi3/arc/ppboot.lds.S
index 93d8b69..09d4077 100644
--- a/arch/gaudi3/arc/ppboot.lds.S
+++ b/arch/gaudi3/arc/ppboot.lds.S
@@ -63,7 +63,7 @@
 	.stack : {
 		. = ALIGN(8);
 		__stack = . ;
-		. = . + 512 ;
+		. = . + 2048 ;
 		__stack_top = . ;
 	} > REGION_STACK
 
